### DISCUSSION TITLE
Redirection Path on Impersonation

### DISCUSSION THREAD

**Discussion by bsormagec** [laravel/nova-issues#3932] [Ideas & Feature Requests] [answered]
Hey, 
When we are impersonating nova it redirects the main URL of the project. I think it needs to be configurable from User Model or somewhere from the config.

Thanks.

**Comment by jaxramus** [comment]
+1 im having the same issue

**Comment by jaxramus** [comment, marked as answer]
I figured out a little workaround by checking impersonation state in my `/` route:

```php
Route::get('/', function (Request $request, ImpersonatesUsers $impersonator) {
    if ($impersonator->impersonating($request)) {
        return redirect('/nova/resources/leads');
    }

    return view('welcome');
});
```

**Comment by crynobone** [comment]
This has been implemented and will be available on the next release.
